### DISCUSSION TITLE
DeSo Avatar Suggestions

### DISCUSSION THREAD

**Discussion by 54rrdeaniel** [deso-protocol/dips#161] [Medium CIPs]
Avatars are a very important tool for self-expression on a social media platform. DeSo avatars are 100px-100px in size. Therefore, Avatars are of very poor quality. For example, small and important details are lost in avatars. It should be 300px-300px instead of 100px-100px.

**Comment by tijno** [comment]
what is impact on blockchain & scaling of doing this?

pfp are stored on chain

**Reply by stet** [reply]
yes and since you can update your pfp, it could be an _attack vector_ to bloat the chain with a loop of updated images which is one reason to keep tham as small as possible. 
maybe a higher fee for profile updates or for image insertion specifically.

**Comment by darian-parrish** [comment]
Similar discussion with more recent engagement: https://github.com/deso-protocol/dips/discussions/174
